styles.css

```diff
@@ -0,0 +1,767 @@
+    gap: 16px;
+    position: relative;
+    top: 16px;
+  }
+
+  .navigation-item {
+    list-style: none;
+    display: flex;
+    justify-items: center;
+    width: 67px;
+  }
+
+  .navigation-item-text {
+    width: 63px;
+    height: 20px;
+    font-family: "inter", serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    text-decoration: none;
+    color: #344563;
+  }
+
+  #toolbar-menu-button {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    position: absolute;
+    width: 40px;
+    height: 40px;
+    right: 24px;
+    top: 2px;
+    border-radius: 8px;
+  }
+
+  #toolbar-menu-icon {
+    width: 24px;
+    height: 24px;
+    position: relative;
+    justify-content: center;
+    align-items: center;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    top: 20px;
+    left: 8px;
+    border: none;
+  }
+
+  #toolbar-menu-union-icon,
+  #toolbar-menu-union-icon::before,
+  #toolbar-menu-union-icon::after {
+    background-color: #28352f;
+    position: absolute;
+    width: 100%;
+    height: 2px;
+  }
+
+  #toolbar-menu-union-icon::before,
+  #toolbar-menu-union-icon::after {
+    content: "";
+  }
+
+  #toolbar-menu-union-icon::before {
+    transform: translate(0, 8px);
+  }
+
+  #toolbar-menu-union-icon::after {
+    transform: translate(0, -8px);
+  }
+
+  /* Headline section */
+  #headline {
+    height: auto;
+    padding: 160px 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 24px;
+  }
+
+  .headline-heading {
+    width: 100%;
+    font-family: 'Inter', serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 260%;
+    line-height: 52px;
+    text-align: center;
+  }
+
+  #color-this {
+    color: green;
+    letter-spacing: 3px;
+  }
+
+  #color-this::before {
+    content: "\a";
+    white-space: pre;
+  }
+
+  .headline-description {
+    width: 87%;
+    font-family: 'Inter', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #172b4d;
+  }
+
+  #headline-social-media {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .social-media {
+    display: flex;
+    gap: 24px;
+  }
+
+  .bottom-icon-container {
+    width: 40px;
+    height: 40px;
+    border: 1px solid #dfe1e6;
+    border-radius: 20px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .headline-to-bottom-icon {
+    height: 114px;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .bottom-icon-container:hover {
+    background-color: #36b37f;
+  }
+
+  /* WORKS SECTION */
+  #works {
+    width: 81%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 80px 0;
+    gap: 48px;
```

Comment:
You did a nice job with the page layout👍 but for this milestone, it is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) that we lay out the entire work section with `grid` while the individual cards are done with `flex` kindly make these adjustments😉😉

<img width="770" alt="css__grid" src="https://user-images.githubusercontent.com/64643995/219516512-223d0e26-15f5-427b-9afe-4fb2e3ffd9e8.png">

Reply:
Fixed.